src/components/app/details/Detail.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Detail = ({ name, photoUrl, allies, enemies, affiliation }) => (
+  <figure>
+    <figcaption>{name}</figcaption>
+    <img src={photoUrl} />
+    <div>Allies: {allies}</div>
+    <div>Enemies: {enemies}</div>
+    <div>Affiliation: {affiliation}</div>
```

Comment:
Not every character has an affiliation so you could do some conditional rendering here! 

https://reactjs.org/docs/conditional-rendering.html